src/styles/components/about-us/flippingCard.module.scss

```diff
@@ -0,0 +1,39 @@
+.container {
+    background-color: (var, --color-beige);
+    border-radius: 12px;
+}
+
+.FlipCard {
```

Comment:
here, you can add an `overflow:hideen` to fix the overflowing content after the above change in the case you remove the `container` div